src/systems/camera.js

```diff
@@ -100,11 +100,15 @@ module.exports.System = registerSystem('camera', {
     cameraEls = sceneEl.querySelectorAll('[camera]');
     for (i = 0; i < cameraEls.length; i++) {
       cameraEl = cameraEls[i];
-      if (newCameraEl === cameraEl) { continue; }
+      if (isMixin(cameraEl) || newCameraEl === cameraEl) { continue; }
```

Comment:
You can do `!cameraEl.isEntity`

Reply:
Will it return `true` for primitives also?

Reply:
Yeah.